### ISSUE TITLE
Migrate custom code

### ISSUE THREAD

**Issue by otakupahp** [a8cteam51/simple-cpt-press#1]
### Description

There is an implement of this, but it does not complain the last code standards, so a migration is needed.

**Comment by otakupahp** [comment]
This is done in another issue

**Comment by otakupahp** [comment]
Hi, @tommusrhodus & @nate-allen.

This is the repo I mentioned at the [monorepo PR](https://github.com/a8cteam51/special-projects-blocks-monorepo/pull/35). 

Do you think is it ok to put the code here?

**Comment by tommusrhodus** [comment]
Yes, just make TAMs aware of its existence and make sure it's linked back to from the project readme where it's in use.

**Comment by otakupahp** [comment]
The code was already updated and pushed to this repo.
